src/components/PupilJourneyComponents/QuizMCQSingleAnswer/QuizMCQSingleAnswer.tsx

```diff
@@ -23,89 +31,44 @@ export const QuizMCQSingleAnswer = (props: QuizMCQSingleAnswerProps) => {
   const questionState = quizEngineContext.questionState[currentQuestionIndex];
   const questionUid = currentQuestionData?.questionUid;
 
-  const lastChanged = useRef<number>(0);
-
-  useEffect(() => {
-    lastChanged.current = 0;
-  }, [currentQuestionIndex]);
-
   if (!questionState) {
     return null;
   }
 
-  const handleOnChange = () => {
-    if (lastChanged.current === 0 && onInitialChange) {
-      onInitialChange();
-    } else if (lastChanged.current !== 0 && onChange) {
-      onChange();
-    }
-    lastChanged.current = Date.now();
-  };
-
   const isFeedbackMode = questionState.mode === "feedback";
 
   return (
     <OakRadioGroup
-      name={questionUid || "quiz"}
+      name={questionUid || "mcq-single-answer"}
       $flexDirection={"column"}
+      $gap={"space-between-s"}
       onChange={handleOnChange}
       disabled={isFeedbackMode}
     >
       {answers?.map((answer, i) => {
-        return (
-          <OakBox key={`${questionUid}-answer-${i}`}>
-            {answer.answer.map((answerItem) => {
-              let feedbackModeColor: "oakGreen" | "red" | undefined;
-
-              if (questionState.feedback?.[i] === "correct") {
-                feedbackModeColor = "oakGreen";
-              } else if (questionState.feedback?.[i] === "incorrect") {
-                feedbackModeColor = "red";
-              }
+        const label = answer.answer.find((a) => a.type === "text") as
+          | StemTextObject
+          | undefined;
```

Comment:
Looks like a job for `isText` if you export it from your new stem utils

```suggestion
        const label = answer.answer.find(isText)
```